### PR TITLE
API-4734-labs-vista-rpc-model

### PR DESCRIPTION
**Pull request by jhulbert-va**
# [API-4734](https://vajira.max.gov/browse/API-4734)

### CODE REVIEW

File: vistalink-models/src/main/java/gov/va/api/lighthouse/vistalink/models/vprgetpatientdata/Labs.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class Labs {
+  @JacksonXmlProperty(isAttribute = true)
+  Integer total;
```
Comment (bschofield-va) [inline vistalink-models/src/main/java/gov/va/api/lighthouse/vistalink/models/vprgetpatientdata/Labs.java:24]: I'm noting this as an observation, not necessarily a request for change.
We've established this pattern of _package_ protected fields for model classes, which is not bestest ever. I'd prefer `private`.
As a general reminder, `@Data` does not change protection of fields like `@Value` does.

File: vistalink-models/src/test/resources/junit-platform.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+junit.jupiter.execution.parallel.enabled = true
```
Comment (bschofield-va) [inline vistalink-models/src/test/resources/junit-platform.properties:1]: interesting... nice

**Review by bschofield-va — APPROVED** [review]
I'm assuming your model is correct /shrug

File: vistalink-models/src/main/java/gov/va/api/lighthouse/vistalink/models/vprgetpatientdata/Labs.java (hunk 1 above)
Comment (jhulbert-va) [reply vistalink-models/src/main/java/gov/va/api/lighthouse/vistalink/models/vprgetpatientdata/Labs.java:24]: Will create a task to go back and do that (looks like it's an issue in Vitals as well). Right now, I'd like to see how far we can push the lab task.

File: vistalink-models/src/test/resources/junit-platform.properties (hunk 2 above)
Comment (jhulbert-va) [reply vistalink-models/src/test/resources/junit-platform.properties:1]: The control can be defined by classes/tests. I wonder if we could parallelize DQ's test classes and not the tests within those classes to gain some speed without running into the JPATest issues.

**Review by eclendenning-va — APPROVED** [review]

**Pull request merged**